container-builder/builder/kubernetes/testdata/Dockerfile

```diff
@@ -1,3 +1,18 @@
+# Copyright 2023 Red Hat, Inc. and/or its affiliates
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+#Not yet released Change the 31 st oct with quay.io/repository/kiegroup/kogito-swf-builder:latest
```

Comment:
Why 31 st oct ?

Reply:
I didn't do this change, it's in my commit 'cause I've moved the file. @radtriste is this yours? Should I change it?

Reply:
I think this was from Max because of builder iamge not updated yet. But yeah can be removed